.github/workflows/unittest-windows-cpu.yml

```diff
@@ -0,0 +1,59 @@
+name: Unittests on Windows CPU
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+jobs:
+  unittests-windows-cpu:
+    uses: pytorch/test-infra/.github/workflows/windows_job.yml@main
+    with:
+      repository: pytorch/audio
+      runner: windows.4xlarge
+      timeout: 180
+      script: |
+        # Mark Build Directory Safe
+        git config --global --add safe.directory /__w/audio/audio
+
+        # Set up Environment Variables
+        export PYTHON_VERSION="3.8"
+        export USE_FFMPEG="1"
+        unset CUDA_VERSION
+
+        # Set CHANNEL
+        if [[(${GITHUB_EVENT_NAME} = 'pull_request' && (${GITHUB_BASE_REF} = 'release'*)) || (${GITHUB_REF} = 'refs/heads/release'*) ]]; then
+          export UPLOAD_CHANNEL=test
+        else
+          export UPLOAD_CHANNEL=nightly
+        fi
+
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CMD_APPLY_CMVN_SLIDING="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CMD_COMPUTE_FBANK_FEATS="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CMD_COMPUTE_KALDI_PITCH_FEATS="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CMD_COMPUTE_MFCC_FEATS="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CMD_COMPUTE_SPECTROGRAM_FEATS="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CMD_SOX="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CUDA="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_HW_ACCEL="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_KALDI="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_SOX="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_ON_PYTHON_310="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_MOD_sentencepiece="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_AUDIO_OUT_DEVICE="1"
+        export TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_MACOS="1"
+
+        # kenlm wheels builds have been broken on Windows for quite some time.
```

Comment:
I think it's more useful to mention why it was broken.
And the reason is because somehow compiling KenLm on Windows stopped working for [a part that is not changed for 10 years](https://github.com/kpu/kenlm/blame/4e6ac85c8d01ac91cb61dfbdc76cd652158c5969/util/read_compressed.cc#L228).

Reply:
Thanks for the added context - I'll append this comment

Reply:
The fixes in https://github.com/pytorch/audio/pull/3378 and https://github.com/pytorch/audio/pull/3381 made the unittests work on trunk again with kenlm. I brought this dependency back in the Nova job as well.